src/ui/edgeDevToolOpener.ts

```diff
@@ -48,7 +48,7 @@ export class EdgeDevToolOpener implements IExtensionContribution {
         }
 
         try {
-          return await vscode.commands.executeCommand(commandId, session.id);
+          return await vscode.commands.executeCommand(commandId, session.id, session.configuration);
```

Comment:
I would much rather pull the root session's configuration by iterating up `session.parentSession` than plumbing select properties down through individual sessions just for this command

Reply:
Thanks! I've made the change. I wasn't sure reaching up to the parentSession would be acceptable in this context.